src/apps/search/panels/BasePanel.tsx

```diff
@@ -155,6 +166,13 @@ const BasePanel = (props: Props) => {
   },
   [data, props.name]);
 
+  /**
+   * Gets the detail page URL for the passed item.
+   */
+  const getItemUrl = useCallback((model, data) => {
+    return `/${lang}/${model}/${data?.uuid}`
+  }, [lang])
```

Comment:
Missing semi-colons.

Reply:
What if I open a separate PR to add ESLint and some sort of basic linting config? It seems time-consuming to go through the files and manually replace quotation marks, etc. I see there's an issue for it already: https://github.com/performant-software/core-data-places/issues/26